README.md

```diff
@@ -159,6 +160,26 @@ A simple example on how to do so, the user must deploy a `ConfigMap`, such as in
 
 Currently, this configuration is not dynamic.
 
+## KubeSecondaryDNS
+
+[This controller](https://github.com/kubevirt/kubesecondarydns)
+allows to support FQDN for VMI's secondary networks.
```

Comment:
only VMI? not VMs?

Reply:
Right, because you must have a running guest, with ip that is reported by the guest agent